lib/hexpm/web/views/layout_view.ex

```diff
@@ -42,9 +42,17 @@ defmodule Hexpm.Web.LayoutView do
       tag(:meta, property: "og:title", content: Map.get(assigns, :title)),
       tag(:meta, property: "og:type", content: "website"),
       tag(:meta, property: "og:url", content: Map.get(assigns, :canonical_url)),
-      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/hex.png")),
+      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/open-graph.png")),
```

Comment:
@pragmaticivan I did some visual testing:

new hex card vs github on slack:

![zrzut ekranu 2017-03-17 o 13 44 51](https://cloud.githubusercontent.com/assets/76071/24043690/25144fca-0b18-11e7-8ef8-c4daeb9194f4.png)

github card on twitter:

![zrzut ekranu 2017-03-17 o 13 49 35](https://cloud.githubusercontent.com/assets/76071/24043809/b878241c-0b18-11e7-9703-f3b19aab2fec.png)


Based on these, I think a smaller logo, just Hex icon, would probably serve us better. Otherwise, the "big" image would be either scaled to be too small or cropped